client/epics/night-mode-epic.js

```diff
@@ -35,12 +35,14 @@ export default function nightModeSaga(
       }
     })
     .filter(() => false);
+
   const toggle = actions
     .filter(({ type }) => types.toggleNightMode === type);
 
+  // Set new theme ('night' or 'default') to local storage
```

Comment:
I don't think this comment is entirely accurate. From what I can tell, this is an observable that, when triggered, gets the current theme, switches it, and calls _a different function_ to store it. This function is not the function that actually sets it to local storage. I think this is better off without a comment.

Reply:
You're right, I'll make the change.